zerver/tests/fixtures/ldap/directory.json

```diff
@@ -87,14 +87,16 @@
         "cn": ["cool_test_group"],
         "uniqueMember": [
             "uid=hamlet,ou=users,dc=zulip,dc=com"
-        ]
+        ],
+        "description": "cool_test_group_description"
     },
     "cn=another_test_group,ou=groups,dc=zulip,dc=com": {
         "objectClass": ["groupOfUniqueNames"],
         "cn": ["another_test_group"],
         "uniqueMember": [
             "uid=hamlet,ou=users,dc=zulip,dc=com",
             "uid=cordelia,ou=users,dc=zulip,dc=com"
-        ]
+        ],
+        "description": "another_test_group_description"
```

Comment:
No I have not tested it with a real LDAP. You are absolutely correct I misunderstood RFC4519, it will be returned in a list. Thanks for pointing that out, the way you showed 
```
description = group_info[1][settings.LDAP_GROUP_DESCRIPTION_ATTR][0]
```

Do you want me to correct this?

Reply:
@beanaas Yup, that would be helpful! It's also pretty important to give this some basic testing with a real LDAP setup, just to be sure end-to-end that this works. Are you able to do such a test? If you have your own self-hosted server where you're losing LDAP, that'd probably be pretty easy by just deploying this pull request there and testing. Otherwise it involves setting up some kind of ldap integration in the development environment. 

If you won't be able to test that, I'll try to find some time to do the development environment set up myself and give this a try; because I think we need to do that before merging.

Reply:
Okey, will fix that. I don't have a real LDAP setup, but I will try to create one and test on that. If I can't manage to fix that I will write to you.

Reply:
Okay sounds good - thank you!

Reply:
@mateuszmandera Now I have tested with a real LDAP server and it worked as you said, sorry for the late response took more time then anticipated setting the LDAP up.